src/pages/management/webhooks/index.mdx

```diff
@@ -27,7 +27,7 @@ In order to receive webhooks, you need to [register](/management/configuration-a
 | **Thread tags**   | [`thread_tagged`](#thread_tagged) [`thread_untagged`](#thread_untagged)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
-| **Status**        | [`routing_status_set`](#routing_status_set) [`agent_deleted`](#agent_deleted)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| **Status**        | [`routing_status_set`](#routing_status_set)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | **Customers**     | [`incoming_customer`](#incoming_customer) [`customer_session_fields_updated`](#customer_session_fields_updated)                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | **Configuration** | [`agent_created`](#agent_created) [`agent_approved`](#agent_approved) [`agent_updated`](#agent_updated) [`agent_suspended`](#agent_suspended) [`agent_unsuspended`](#agent_unsuspended) [`agent_deleted`](#agent_deleted) [`auto_access_added`](#auto_access_added) [`auto_access_updated`](#auto_access_updated) [`auto_access_deleted`](#auto_access_deleted) [`bot_created`](#bot_created) [`bot_updated`](#bot_updated) [`bot_deleted`](#bot_deleted) [`group_created`](#group_created) [`group_updated`](#group_updated) [`group_deleted`](#group_deleted) |
```

Comment:
I agree that if the description is old,  then there is no need to keep it :)
One question - should we also remove it from this line?

Reply:
For me it's better to remove the `agent_deleted` from `Status` section, because all `agent_*` webhooks are now in `Configuration` section